src/Agent.Worker/Worker.cs

```diff
@@ -189,7 +190,7 @@ private void InitializeSecretMasker(Pipelines.AgentJobRequestMessage message)
                 {
                     if (!string.IsNullOrEmpty(keyValuePair.Value) && MaskingUtil.IsEndpointAuthorizationParametersSecret(keyValuePair.Key))
                     {
-                        HostContext.SecretMasker.AddValue(keyValuePair.Value);
+                        HostContext.SecretMasker.AddValue(keyValuePair.Value, keyValuePair.Key);
```

Comment:
Could it make sense to specify in message here also - that it comes as service endpoint field value? I think only key value does not make much more verbose

Reply:
added